tests/siuparser.test.js

```diff
@@ -0,0 +1,63 @@
+import { siuAxel, siuFede, siuExactas } from "./siu-files";
+import { parseSIU } from "../src/siuparser";
+
+const sius = [ siuAxel, siuFede, siuExactas ];
+
+describe.each(sius)("essential tests", (siuName, siuRawData, siuJSON) => {
+  const parsedSIU = parseSIU(siuRawData);
+
+  test(`${siuName} parsed siu does not change`, () => {
+    siuJSON.forEach((json, index) => {
+      json.timestamp = parsedSIU[index].timestamp;
+    });
+    expect(parsedSIU).toEqual(siuJSON);
+  });
+
+  test(`${siuName} periodo is a string`, () => {
```

Comment:
Que pensas de cambiar este a `periodo is a non-empty string` (y el codigo para chequear la length de la cadena) para asegurarnos una cosita extra